libvast/src/experimental_table_slice.cpp

```diff
@@ -276,6 +274,18 @@ auto decode(const type& t, const arrow::Array& arr, F& f) ->
       using array_type = arrow::NumericArray<arrow::UInt64Type>;
       return dispatch(static_cast<const array_type&>(arr));
     }
+    case arrow::Type::EXTENSION: {
```

Comment:
More generally: Does it make sense to specialize `caf::sum_type_access` such that we can use `caf::get<T>`, `caf::get_if<T>`, `caf::holds_alternative<T>`, and `caf::visit` with Arrow Array and Arrow DataType?

Reply:
will be handled in a separate PR / issue.